semcore/popper/src/Popper.jsx

```diff
@@ -147,6 +148,7 @@ class Popper extends Component {
   }
 
   state = { ignoreTriggerFocusUntil: 0 };
+  mouseEnterCursorPositionRef = { current: null };
```

Comment:
I'm not sure, that you need ref here... you could just use object, as I see...

Reply:
I would prefer to keep normal ref object just to make it more predictable